turbinia/config/turbinia_config_local.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
```

Comment:
After taking a cursory look at this, it seems like there aren't too many changes from the GCP version of the config.   Rather than making a copy of this, could we potentially update the existing one to make it more suitable to be both?   I'd prefer not to have to update things in multiple places if it's not needed.  Maybe we could have a commented out line for each of the lines that change to make it easier for people to switch?  I'd have to double check, but I think that Turbinia won't load anything Redis/Celery related unless the state_manager/task_manager vars are configured to do so, so we could potentially also just change the REDIS_* and CELERY_* vars in the main config to be what they are here, and I think the only difference would be the state_manager/task_manager vars.

Reply:
Removed the local file and updated the REDIS_*/CELERY_*. It might be better to specify which fields need to be updated for local setup.